src/EditorFeatures/Core/EditorFeatures.csproj

```diff
@@ -107,6 +108,7 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.InteractiveEditorFeatures" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.CSharp.Repl" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.InteractiveServices" />
+    <InternalsVisibleTo Include="Microsoft.VisualStudio.InteractiveWindow" />
```

Comment:
This layering is backwards, I believe. Roslyn shouldn't be granting code to the general support.